docs/en/ufs/HDFS.md

```diff
@@ -31,8 +31,8 @@ directory:
 $ mvn install -P<YOUR_HADOOP_PROFILE> -D<HADOOP_VERSION> -DskipTests
 ```
 
-Alluxio provides predefined build profiles including `hadoop-1`, `hadoop-2` (enabled by default),
```

Comment:
these are still true, right? are we also removing the build profile for hadoop 1?

Reply:
I'll raise a separate PR to remove the hadoop-1 profile

Reply:
oh shall we remove them from alluxio codebase? I think we can still keep them here.

Reply:
I don't think we want to remove the `hadoop-1` profile, otherwise users cannot build `hadoop-1` by themselves?